OJP/OJP_Trips.xsd

```diff
@@ -575,7 +575,7 @@
 	</xs:simpleType>
 	<xs:complexType name="LegStructure">
 		<xs:annotation>
-			<xs:documentation>a single stage of a TRIP that is made without change of MODE or service (ie: between each interchange)</xs:documentation>
+			<xs:documentation>A single stage of a TRIP that is made without change of MODE or service (ie: between each interchange). Implements LEG from TM 6.</xs:documentation>
```

Comment:
This is the only place where you mention TM 6, instead of TM 6.2. Is this correct?

Reply:
I think it was a LEG already in TM 6, so I think it is ok.